### DISCUSSION TITLE
assign to group or specific user notification automatically

### DISCUSSION THREAD

**Discussion by alexeysvyridov** [WeblateOrg/weblate#5191] [Q&A] [answered]
I didn't found information how assign to group or translator notification **automatically**.
Is it possible to do ?
Only user can apply this option manual -> Languages -> notifications: "project name"
If i understand correct only admin will be getting notifications by default (duplicates and c.t.r) 
**Explanation**: i want assign to translator, which will be get a  notifications to email ("duplicates ")

**Comment by nijel** [comment, marked as answer]
It's currently not easily possible, see https://github.com/WeblateOrg/weblate/issues/5155

**Reply by alexeysvyridov** [reply]
thank you
